config/routes.rb

```diff
@@ -6,12 +6,16 @@
 Rails.application.routes.draw do
   mount Rswag::Ui::Engine => '/api-docs'
   mount Rswag::Api::Engine => '/api-docs'
-  devise_for :users
+  
+  devise_for :users, controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
+  }
 
-  namespace :api do
-    namespace :v1 do
-      resources :users do
-        resources :expertises
+    namespace :api do
+       namespace :v1 do
+        resources :users do
+          resources :expertises
```

Comment:
Unnecessary indentation change.

Reply:
Good work with setting up jwt authentication, it seems like everything is working! However, it looks like doing so broke a lot of specs, we'll need to set up some kind of mock authentication on that end.